exocom-server/bin/spec

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-set -e
+set -ex
```

Comment:
This isn't run on CI, only on developer machines. I'd like to keep the noise that this change introduces away from developers. There was never an issue with a developer not knowing what exact command failed.